webapp/locales/en.json

```diff
@@ -25,7 +25,16 @@
     "shouted": "Shouted",
     "commented": "Commented",
     "userAnonym": "Anonymous",
-    "socialMedia": "Where else can I find"
+    "socialMedia": "Where else can I find",
+    "network": {
+      "title": "Network",
+      "following": "is following:",
+      "followingNobody": "follows nobody.",
+      "followedBy": "is followed by:",
+      "followedByNobody": "is followed by nobody.",
```

Comment:
this would be better `is not followed by anyone`

Reply:
Thanks for the commit!